modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
-| *Infrastructure* a|
-
-* Application Process CPU Load: Amount of CPU load used by the JVM
-* Application Process CPU Minutes: CPU time used by the process on which the JVM is running
-* Operating System Swap Space: Amount of memory the system is swapping
-* CPU Utilization %: Percentage of CPU used by each worker over time
-* Memory Utilization: Amount of memory used by each worker over time
-* Total System Processors: Number of system processors for workers available over time
-* Total System Memory: Amount of system memory available for workers over time. +
-If you have a Titanium subscription and the selected application is hybrid, this graph changes to *System Memory*.
-* System Memory: This graph displays only if you have a Titanium subscription and the selected application is hybrid. This graph displays the total free system memory per worker.
-* Thread Count - Server: Number of simultaneous requests for a worker over time
+* All Inbound Requests Grouped by Response Type
+* All Outbound Requests Grouped by Response Type
+* Failed Inbound Requests: Response Time by Endpoint
+* Total Failed Inbound Requests Grouped by Endpoint
+* Failed Outbound Requests: Response Time by HTTP Endpoint
+* Total Failed Outbound Requests Grouped by HTTP Endpoint
+| <<jvm-charts>> a|
+
+* Garbage Collection Count 
+* Garbage collection Time
+* Par new Collection Count
+* Par new collection Time
+* Classes (Loaded, Loaded Total, Unloaded)
+* Heap (Committed, Used)
+* Thread Count
+* JVM Uptime
+* Par Eden graphs (Used, Max, Init, and Committed)
+* Par Survivor graphs (Used, Max, Init, and Committed)
+* Metaspace graphs (Used, Max, Init, Committed)
+* Code Cache graphs (Used, Max, Init, Committed)
+* Compressed Class Space graphs (Used, Max, Init, Committed)
+* Committed Virtual Memory
+| <<infrastructure-charts>> a|
+
+* Application Process CPU Load
+* Application Process CPU Minutes
+* Operating System Swap Space
+* CPU Utilization %
+* Memory Utilization
+* Total System Processors:
+* Total System Memory
+* System Memory
+* Thread Count - Server
+
+| <<custom-metrics-charts>> a|
+
+The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
+|===
+
+[[overview-charts]]
+== Overview
+
+The following charts appear on the *Overview* dashboard:
+
+Inbound - Total Requests by Response Type::
+The sum of inbound requests, grouped by type of response in automatic intervals. Responses are OK or Failed. If a Mule app has more than one connector type, all types are included in this metric.
+
+Inbound - Average Response Time::
+The average response time of inbound requests, grouped by application in automatic intervals. If a Mule app has more than one connector type, all types are included in this metric.
+
+Mule Messages::
+The sum of Mule messages that are processed successfully, grouped by application.
```

Comment:
Remove "grouped by application". This dashboard is for a single application only.